src/maincurrencydialog.cpp

```diff
@@ -663,8 +660,7 @@ void mmMainCurrencyDialog::OnHistoryUpdate(wxCommandEvent& /*event*/)
         {
             for (auto &entry : historical_rates)
             {
-                Model_CurrencyHistory::instance().addUpdate(m_currency_id
-                    , entry.first, entry.second, Model_CurrencyHistory::ONLINE);
+                Model_CurrencyHistory::instance().addUpdate(m_currency_id, entry.first, entry.second, Model_CurrencyHistory::ONLINE);
```

Comment:
Too long lines is bad. It decrease a code redability.

Reply:
Up to me it's the opposite: I don't like code that could stay in one line and instead it's divided into two/three. Generally I try to keep it inline if it stays in one line on my FullHD monitor.

Reply:
Screens are increasing in resolution, if you keep on one line code that could stay on one line, then you need less scrolling to see all the cpp code